cratedb_toolkit/api/main.py

```diff
@@ -119,14 +119,24 @@ def load_table(self, resource: InputOutputResource, target: TableAddress, transf
         if source_url_obj.scheme.startswith("dynamodb"):
             from cratedb_toolkit.io.dynamodb.api import dynamodb_copy
 
-            if not dynamodb_copy(str(source_url_obj), target_url, progress=True):
-                msg = "Data loading failed"
-                logger.error(msg)
-                raise OperationFailed(msg)
+            if dynamodb_copy(str(source_url_obj), target_url, progress=True):
+                return True
+            else:
+                logger.error("Data loading failed or incomplete")
+                return False
 
         elif source_url_obj.scheme.startswith("file"):
             if "+bson" in source_url_obj.scheme or "+mongodb" in source_url_obj.scheme:
-                mongodb_copy_generic(
+                return mongodb_copy_generic(
+                    str(source_url_obj),
+                    target_url,
+                    transformation=transformation,
+                    progress=True,
+                )
+
+        elif source_url_obj.scheme.startswith("http"):
```

Comment:
Maybe move this up to `if source_url_obj.scheme.startswith("file") or source_url_obj.scheme.startswith("http")` as the body of both branches looks the same?

Reply:
Significantly improved with fb70a36953. Thanks!